test/semantics/dosemantics09.f90

```diff
@@ -0,0 +1,125 @@
+! Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+
+!C1129 
+!A variable that is referenced by the scalar-mask-expr of a
+!concurrent-header or by any concurrent-limit or concurrent-step in that
+!concurrent-header shall not appear in a LOCAL locality-spec in the same DO
+!CONCURRENT statement.
+
+subroutine s1()
+
+!ERROR: 'i' is already declared in this scoping unit
+  do concurrent (i=1:10) local(i)
+  end do
+end subroutine s1
+
+subroutine s2()
+!ERROR: 'i' is already declared in this scoping unit
+  do concurrent (i=1:10) local_init(i)
+  end do
+end subroutine s2
+
+subroutine s3()
+!ERROR: 'i' is already declared in this scoping unit
+  do concurrent (i=i:10) shared(i)
```

Comment:
You meant `i=i:10` or `i=1:10`?

Reply:
Good catch!  This is just a bogus extra test.  I'll get rid of it.